components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/model/AuthenticatedUser.java

```diff
@@ -72,6 +72,9 @@ public AuthenticatedUser(AuthenticatedUser authenticatedUser) {
             this.userAttributes.putAll(authenticatedUser.getUserAttributes());
         }
         this.isFederatedUser = authenticatedUser.isFederatedUser();
+        if (!isFederatedUser && StringUtils.isNotEmpty(userStoreDomain) && StringUtils.isNotEmpty(tenantDomain)) {
```

Comment:
Here we are checking userStoreDomain & tenantDomain with StringUtils.isNotEmpty but in https://github.com/wso2/carbon-identity-framework/pull/1782/files#diff-9a1a18501271ed0bfdd697758de171a6R94 only checking null (userStoreDomain != null, here empty string checking is missing). Don't these two places should be same?

Reply:
+1. thanks for pointing that out.